test/utils/jasmine-matchers.js

```diff
@@ -122,6 +123,36 @@ exports.execute = function() {
                         };
                     }
                 }
+            },
+
+            toThrow: function(util, customEqualityTesters) {
```

Comment:
Why did you add your own `toThrow` matcher? Does it override Jasmine's native `toThrow` matcher? What's the difference between this implementation of `toThrow` with the native `toThrowError`?